db-commons/src/main/scala/org/bitcoins/db/AppConfig.scala

```diff
@@ -73,7 +70,7 @@ abstract class AppConfig extends LoggerConfig with StartStopAsync[Unit] {
 
     val numOverrides = configs.length + 1
 
-    if (logger.isDebugEnabled()) {
+    if (logger.logger.isDebugEnabled()) {
```

Comment:
I think all of these can be simplified too: 

```scala
logger.isDebugEnabled
```

but i'm suspicious that we don't even needs these anymore?

Reply:
Sadly it can't be simplified. Also we do still want them because we have some computation in these brackets that we would like to skip if we can.